### DISCUSSION TITLE
Brick wallet como modal

### DISCUSSION THREAD

**Discussion by albertoammar** [mercadopago/sdk-js#206] [Q&A] [answered]
Como saber se o cliente já finalizou o pagamento ? Gostaria de exibir uma pagina sucesso após a finalização do pagamento por esse metodo (modal):

https://www.mercadopago.com.br/developers/pt/docs/checkout-api/integration-configuration/integrate-mp-wallet

**Comment by icaldana** [comment, marked as answer]
Olá @albertoammar! 

Obrigado por utilizar Mercado Pago!
Te recomendo dar uma olhada [nessa documentação](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/wallet-brick/introduction) de integração com Wallet Button, com a qual é possível configurar [URLs de retorno](https://www.mercadopago.com.br/developers/pt/docs/checkout-bricks/wallet-brick/advanced-features/preferences) explicadas no final da página. 

Espero que essas informações te auxiliem 😄

**Reply by albertoammar** [reply]
Obrigado @icaldana estava tão crente que o "problema" precisa ser resolvido no front que ignorei o back. Obrigado deu certo aqui haha
